src/script/page/LeftSidebar/panels/StartUI/components/groupList/GroupListItem.tsx

```diff
@@ -43,8 +43,9 @@ const GroupListItem: React.FC<GroupListItemProps> = ({click, group}) => {
   } = useKoSubscribableChildren(group, ['display_name', 'participating_user_ets', 'is1to1']);
 
   const onClick = () => {
+    const url = generateConversationUrl(group.qualifiedId);
     click(group);
-    navigate(generateConversationUrl(group.qualifiedId));
+    createNavigate(url);
```

Comment:
This change seems suspicious to me. You create something but never return it, this function is never consumed 🤔